README.md

```diff
@@ -18,7 +18,7 @@ We try to avoid breaking changes and the associated major version bumps in this
 
 Suppose you just released version `5.0.1`, a small styling fix in the checkbox widget, for a story you're working on. If the project you're working in currently pulls in `noredink-ui` at version `4.x`, then getting to your styling fix means pulling in a new major version of `noredink-ui`. This breaks all `TextArea` widgets across the project, so those will need to be fixed before you can do anything else, potentially a big effort.
 
-To prevent these big Yaks from suddenly showing up in seemingly trivial tasks we prefer to avoid breaking changes in the package. Instead when we need to make a breaking change in a widget, we create a new module for it `Nri.Ui.MyWidget.VX`.
+To prevent these big Yaks from suddenly showing up in seemingly trivial tasks we prefer to avoid breaking changes in the package. Instead when we need to make a breaking change in a widget, we create a new module for it `Nri.Ui.MyWidget.VX`. Similarly, when we build custom elements in JavaScript we create a file `lib/MyWidget/VX.js` and define a custom element `nri-mywidget-vX`.
```

Comment:
Read me to-do discussed IRL: add line about `rm styleguide-app/assets/javascript.js` to force a rebuild!

Reply:
Can be added to the `make clean` target as well: https://github.com/NoRedInk/noredink-ui/blob/15adb4bd2a517e8f7b7c27f851128eeba9b35e64/Makefile#L18-L19

(`make clean` will force a rebuild of everything though, as opposed to just the custom elements)

That reminds me: if we're checking in the build artifact, should we add a CI check that confirms `javascript.js` is up-to-date? something like what our internal build script does:

```sh
if git status --porcelain | grep . ; then
    echo "your build left non-gitignored trash behind"
    git diff --no-color
    exit 1
fi
```

Reply:
Ooh, that sounds like a good idea!

Reply:
I actually think this artifact is fine to gitignore, and that seems like the least complicated way forward.